src/main/resources/openstack/resources/resources.yaml

```
@@ -77,13 +110,18 @@ node_types:
     properties:
       image:
         type: string
-        description: The Openstack VM Image ID either this property or the 'imageName' property is required
+        description: >
+          Openstack VM Image ID, this property is required when 'imageName' is not set and not booting from a volume
         required: false
       imageName:
         type: string
         description: >
-          The Openstack VM Image Name either this property or the 'image' property is required. This property can be used only if the image
-          name is unique.
+          Openstack VM Image Name, this property is required when 'image' is not set and not booting from a volume
```

```suggestion
          Openstack VM Image Name, this property is required when 'image' is not set and not booting from a volume. This property can be used only if the image name is unique.
```
Still true isn't it?

---

Not described in terraform openstack provider documentation, but yes looking at the implementation, it calls this function https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/images/requests.go#L74 returning an error if more than one imahe is found with this name.  Will add this in the description.